FrontEndReact/src/View/Admin/Add/ImportTasks/AdminImportAssessmentTasks.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+import '../AddUsers/addStyles.css';
+import CourseDropdown from './CourseDropdown';
+import validator from "validator";
+import ErrorMessage from '../../../Error/ErrorMessage';
+
+class AdminImportAssessmentTask extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            error: null,
+            errorMessage: null,
+            validMessage: "",
+            courses: [],
+            selectedCourse: ''
+        }
+        this.setSelectedCourse = (newSelectedCourse) => {
+            this.setState({
+                selectedCourse: newSelectedCourse
+            });
+        };
+    }
+    componentDidMount() {
+        document.getElementById("importAssessmentTasks").addEventListener("click", () => {
+            var message = "Invalid Form: ";
+            if(!validator.isNumeric(String(this.state.selectedCourse)) && validator.equals(this.state.selectedCourse, '')) {
+                message += "Missing Course!";
+            }
+            if(message === "Invalid Form: ") {
```

Comment:
I know this is the standard method in the front end that's been used for checking if a form is invalid, but I really don't like it. The intention isn't clear (a valid form happens when the message is "Invalid") and it's not great practice to use string comparison for this sort of logic.

Reply:
Fixed! I have create a variable called success to clearly indicate whether an error has occurred with form validation!